process/templates/WG_NAME_incubating_stage.md

```diff
@@ -1,21 +1,43 @@
 ## Working Group incubation application
 
-### List of regular contributors
-The WG must have a minimum of 5 contributors from at least 3 different organizations attending regulary.
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 1 Chair
 
-  * "name, affiliation, GitHub ID"
+  * "name, affiliation, GitHUb ID"
 
-### TAC sponsor
-The WG must have a TAC sponsor:
 
+### Working Group (WG) has met all Sandbox requirement
+  * "link to sandbox PR"
+
+### List of regular contributors
+The WG must have a minimum of 5 contributors from at least 3 different organizations attending regularly.
   * "name, affiliation, GitHub ID"
 
+
 ### Mission of the Working Group
 The WG must have a charter or mission statement for review by TAC
-
   * Link to the WG charter or mission statement defining its goals.
 
-### Working Group activity
-The WG must have met at least 5 times
-
+### Governance
+Working Groups must have documented, initial WG governance
+  * "link to governance documents/Charter"
+WG must have met publicly at least 5 times
    * Link to public meeting notes (or ideally recordings)
+WG must have a TAC sponsor
+  * "name of TAC sponsor to assist TI"
+WG must have defined Contributor Guide
+  * "link to contributor guide"
+WG has defined group governance
+  * "link to charter or other document describe how group is managed"
+
+ Reference              | URL |
+|-----------------------|-----|
+| Repo                  |     |
+| Meeting Agenda        |     |
+| OSSF Calendar Entry   |     |
+| Website               |     |
+| Contributing guide    |     |
+| Security.md           |     |
+| Roadmap               |     |
```

Comment:
```suggestion
```

I don't think this is required in any of our docs, although it might make sense at the Graduated stage? I think having a roadmap for Incubating might be a bit early.